src/themes/dspace/app/shared/search-form/search-form.component.scss

```diff
@@ -0,0 +1,3 @@
+.form-control {
+    padding: "8px";
```

Comment:
1) CSS doesn't work --> remove quotes
2) try to use relative or variable-controlled measure